docs/src/solvers/nonlinear_system_solvers.md

```diff
@@ -0,0 +1,163 @@
+
+### SimpleNonlinearSolve.jl
+
+These methods are included with NonlinearSolve.jl by default, though SimpleNonlinearSolve.jl
+can be used directly to reduce dependencies and improve load times.
+SimpleNonlinearSolve.jl's methods excel at small problems and problems defined with static
+arrays.
+
+  - [`SimpleNewtonRaphson()`](@ref): A simplified implementation of the Newton-Raphson
+    method.
+  - [`SimpleBroyden()`](@ref): The classic Broyden's quasi-Newton method.
+  - [`SimpleLimitedMemoryBroyden()`](@ref): A low-memory Broyden implementation, similar to
+    L-BFGS. This method is common in machine learning contexts but is known to be unstable
+    in comparison to many other choices.
+  - [`SimpleKlement()`](@ref): A quasi-Newton method due to Klement. It's supposed to be
+    more efficient than Broyden's method, and it seems to be in the cases that have been
+    tried, but more benchmarking is required.
+  - [`SimpleTrustRegion()`](@ref): A dogleg trust-region Newton method. Improved globalizing
+    stability for more robust fitting over basic Newton methods, though potentially with a
+    cost.
+  - [`SimpleDFSane()`](@ref): A low-overhead implementation of the df-sane method for
+    solving large-scale nonlinear systems of equations.
+  - [`SimpleHalley()`](@ref): A low-overhead implementation of the Halley method. This is a
+    higher order method and thus can converge faster to low tolerances than a Newton method.
+    Requires higher order derivatives, so best used when automatic differentiation is
+    available.
+
+!!! note
+    
+    When used with certain types for the states `u` such as a `Number` or `StaticArray`,
+    these solvers are very efficient and non-allocating. These implementations are thus
+    well-suited for small systems of equations.
+
+### SteadyStateDiffEq.jl
+
+SteadyStateDiffEq.jl uses ODE solvers to iteratively approach the steady state. It is a
+very stable method for solving nonlinear systems with stable equilibrium points, though
+often more computationally expensive than direct methods.
+
+  - [`DynamicSS()`](@ref): Uses an ODE solver to find the steady state. Automatically
+    terminates when close to the steady state.
+  - [`SSRootfind()`](@ref): Uses a NonlinearSolve compatible solver to find the steady
+    state.
+
+### NLsolve.jl
+
+This is a wrapper package for importing solvers from NLsolve.jl into the SciML interface.
+
+  - [`NLsolveJL()`](@ref): A wrapper for
+    [NLsolve.jl](https://github.com/JuliaNLSolvers/NLsolve.jl)
+
+Submethod choices for this algorithm include:
+
+  - `:anderson`: Anderson-accelerated fixed-point iteration
+  - `:newton`: Classical Newton method with an optional line search
+  - `:trust_region`: Trust region Newton method (the default choice)
+
+### MINPACK.jl
+
+MINPACK.jl methods are good for medium-sized nonlinear solves. It does not scale due to
+the lack of sparse Jacobian support, though the methods are very robust and stable.
```

Comment:
Are they? The algorithms aren't showing that in comparison 😅

Reply:
Yeah. not really. Changed the text.